dev-util/sysdig/files/sysdig-0.27.1-luajit-2.1.patch

```diff
@@ -0,0 +1,93 @@
+From 1147b9a4f6f1f9e97b43735e7980da387837de4a Mon Sep 17 00:00:00 2001
```

Comment:
Could you put a link to the relevant Gentoo bug(s) at the top of the patch + where the commit is from?

Reply:
Makes sense - done.